learning/tour-of-beam/backend/go.mod

```diff
@@ -0,0 +1,60 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+module beam.apache.org/learning/tour-of-beam/backend
+
+go 1.18
+
+require (
+	github.com/GoogleCloudPlatform/functions-framework-go v1.5.3
+	gopkg.in/yaml.v3 v3.0.1
+)
+
+require (
+	cloud.google.com/go v0.102.1 // indirect
```

Comment:
It looks that you should run "mod tidy" again because this command, as i know, groups dependencies into direct and indirect

Reply:
it is a native go.mod for Go 1.18
go mod help tidy
```
(Go versions 1.17 and higher retain more requirements in order to
support lazy module loading.)```

Reply:
I changed it to be 1.16
and these have gone away